drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/_report_list.haml

```diff
@@ -1,7 +1,7 @@
 .table-utilities
 %p= page_entries_info @reports, entry_name: _('scorecard')
-.row
-  .col-sm-12
+- if @reports.exists?
+  .row
```

Comment:
I don't think you need row here if there are no columns within it. (could be wrong)

Reply:
You are right, I removed it

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2134/commits/853ff66bf68be3756a0984e22e18909ff23a6f44 switches to Pagy, which also handles showing "no results"